.github/dependabot.yml

```diff
@@ -3,7 +3,7 @@ updates:
   - package-ecosystem: "cargo"
     directory: "/"
     schedule:
-      interval: "daily"
+      interval: "weekly"
```

Comment:
A downside of this is that VRL updates will be pulled in less frequently. We could create two separate entries here for `cargo` and update patches weekly, but everything else dailly (or at least VRL).

Reply:
Do you know how to do this?  

```
version: 2
updates:
  - package-ecosystem: "cargo"
    directory: "/"
    schedule:
      interval: "daily"
    allow:
      - dependency-name: "vrl"
  - package-ecosystem: "cargo"
    directory: "/"
    schedule:
      interval: "weekly"
      time: "04:00" # UTC
    labels:
      - "domain: deps"
      - "no-changelog"
    commit-message:
      prefix: "chore(deps)"
    open-pull-requests-limit: 100
    groups:
      patches:
        applies-to: version-updates
        patterns:
        - "*"
        update-types:
        - "patch"
...
```

As always testing this is, requires a bit of trial error.

Reply:
I think you'll need to duplicate all of the fields (like `labels`, `commit-message`, etc.). Maybe YAML merges can be used to avoid some of the duplication?

Reply:
You'll also want to exclude `vrl` from the second set.

Reply:
Hmm https://github.com/vectordotdev/vector/runs/33863412120

Reply:
On a second thought, I think we can live with weekly VRL updates. But if you have a workaround for the above, I am happy to look into it.

Reply:
:+1: it looks like being able to split up updates for a given ecosystem into different schedules is an open feature request: https://github.com/dependabot/dependabot-core/issues/1778

Given that, I'm ok with trying out weekly updates. Do you want to update this PR?

Reply:
Yes, updated.